source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Drupal 7 Long-Term Support now available on Pantheon
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+[Pantheon's Long Term Support (LTS) for Drupal 7](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites) is now available. The default [Pantheon Drupal 7 upstream](https://github.com/pantheon-systems/drops-7) has been updated to include [the Tag1 Drupal 7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es). This module is essential for leveraging our ongoing LTS offering.
+
+### What's new
+
+With this update, future LTS releases for Drupal 7 Core and the D7ES module will be provided through Pantheon’s Drupal 7 upstream. For details, see our [Drupal 7 LTS documentation](/supported-drupal#d7-core-lts-and-platform-compatibility).
+
+### Action required
+
+To continue receiving security coverage for contributed modules:
+
+* Apply the latest upstream update to your Drupal 7 site.
+* Enable the Tag1 D7ES module in your environment.
```

Comment:
We should also add a bullet here instructing the user to configure a notification email address from the Tag1 D7ES configuration page. The config page is at `/admin/config/system/tag1-d7es`

Reply:
It'd be better for us to point folks to docs rather than include all the steps in the release note. Otherwise we'll have to maintain this release note if anything changes.

Reply:
Agree that change should go in the docs. This PR includes both, however, so it can be updated at the same time.